Modules/Bridge/NumPy/wrapping/PyBuffer.i.in

```diff
@@ -77,14 +74,10 @@
         C-order indexing, i.e. k,j,i, the image Size will have the dimensions
         reversed from the array shape.
 
-        Therefore, since the *numpy.transpose* operator on a 2D array simply
+        Therefore, since the *np.transpose* operator on a 2D array simply
```

Comment:
Was this change intentional? `numpy` is more descriptive than `np`.

Reply:
Yes, also update this, but I missed one spot -- adding a  follow-up. In the Python community `np` is standard practice.